data/sql/customizations.sql

```diff
@@ -35,4 +35,20 @@ INSERT INTO authority_to_authority_preference(source_auth_name, target_auth_name
     ('IGNF', 'EPSG', 'PROJ,IGNF,EPSG' );
 
 INSERT INTO authority_to_authority_preference(source_auth_name, target_auth_name, allowed_authorities) VALUES
-    ('ESRI', 'EPSG', 'PROJ,ESRI,EPSG' );
\ No newline at end of file
+    ('ESRI', 'EPSG', 'PROJ,ESRI,EPSG' );
+
+-- Custom ellipsoids (from proj -le)
+
+INSERT INTO "ellipsoid" VALUES('PROJ','ANDREA','Andrae 1876 (Den., Iclnd.)',NULL,'PROJ','EARTH',6377104.43,'EPSG','9001',300.0,NULL,0);
+INSERT INTO "ellipsoid" VALUES('PROJ','CPM','Comm. des Poids et Mesures 1799',NULL,'PROJ','EARTH',6375738.7,'EPSG','9001',334.29,NULL,0);
+INSERT INTO "ellipsoid" VALUES('PROJ','DELMBR','Delambre 1810 (Belgium)',NULL,'PROJ','EARTH',6376428.0,'EPSG','9001',311.5,NULL,0);
+INSERT INTO "ellipsoid" VALUES('PROJ','KAULA','Kaula 1961',NULL,'PROJ','EARTH',6378163.0,'EPSG','9001',298.24,NULL,0);
+INSERT INTO "ellipsoid" VALUES('PROJ','LERCH','Lerch 1979',NULL,'PROJ','EARTH',6378139.0,'EPSG','9001',298.257,NULL,0);
+INSERT INTO "ellipsoid" VALUES('PROJ','MERIT','MERIT 1983',NULL,'PROJ','EARTH',6378137.0,'EPSG','9001',298.257,NULL,0);
+INSERT INTO "ellipsoid" VALUES('PROJ','MPRTS','Maupertius 1738',NULL,'PROJ','EARTH',6397300.0,'EPSG','9001',191.0,NULL,0);
+INSERT INTO "ellipsoid" VALUES('PROJ','NEW_INTL','New International 1967',NULL,'PROJ','EARTH',6378157.5,'EPSG','9001',NULL,6356772.2,0);
+INSERT INTO "ellipsoid" VALUES('PROJ','WGS60','WGS 60',NULL,'PROJ','EARTH',6378165.0,'EPSG','9001',298.3,NULL,0);
+
+-- Custom ellipsoids (ported from QGIS - original source unknown)
```

Comment:
According to https://github.com/qgis/QGIS/blame/master/resources/iau2000.sql , that would come from the IAU2000 dictionary.
@thareUSGS I guess IAU2000 will be injected some time into the PROJ database ?
In the mean time, we can probably keep it under the PROJ authority, and remove it once IAU2000 is integrated.

Reply:
@nyalldawson Just point that it comes from the IAU2000 dictionary. (you may also entirely remove that entry since I doubt anyone would use such a generic ellipsoid for Earth)